.github/workflows/release.yml

```
@@ -0,0 +1,88 @@
+name: Create a New Release
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description:
+          'The version type of this release. Must be either "feature" or "bug", representing a new feature release or a bug fix release, respectively,'
+        required: true
```

```suggestion
      type:
        description: The type of release being performed
        required: true
        type: choice
        options: [feature, bug]
```

https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

---

Note that you'll have to change all the `type` references if you go with the new name here